x/fees/client/cli/query.go

```diff
@@ -70,15 +71,15 @@ func GetCmdQueryFees() *cobra.Command {
 	return cmd
 }
 
-// GetCmdQueryFee implements a command to return a registered contract
-// for fee distribution
+// GetCmdQueryFee implements a command to return a registered contract for fee
+// distribution
 func GetCmdQueryFee() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "fee-info [address]",
+		Use:     "fee [address]",
```

Comment:
In my opinion it would be clearer to have the `Use` be `contract [address]`?
Then a user would be typing

```
evmosd query fees contract [address]
```

This directly implies, that the wanted address is indeed a contract address, as opposed to

```
evmosd query fees fee [address]
```

Reply:
@MalteHerrmann I agree that the command looks nicer. However, shouldn't the query be named after what the query submitter is looking for? In this case, the query returns a `fee` and not a contract.

Reply:
It's true, that it is good practice to name the query after its actual return value and the internal functions in the Go code should definitely be named accordingly. On the other hand, this is the interface to the user and the command `evmosd query fees fee` would mean, that a user would have to know that there even is a `Fee` type and what that means.

As a user, unfamiliar with the codebase and its types, I would simply want to *query* the *feesplit* information for the *contract* I registered, so 

```
evmosd query feesplit contract [address]
```

is more intuitive to me. 
I think including the internal type of the retrieved information in the actual user command, would be a case of developing more for developers instead of focussing on the user experience. What do you think?

In any case, I think the change you have proposed would be an improvement also!

Reply:
@MalteHerrmann Although I like this suggestion for this specific command, I have decided to stick with 

```
evmosd query feesplit fee-split [contract-address]
```

To use the `feesplit` module, developers will have to know anyways what a fee split is and it's more consistent with the msg names.